fabricio/utils.py

```diff
@@ -70,6 +70,9 @@ def make_options(self):
         for option, value in self.items():
             if value is None:
                 continue
+            if isinstance(value, basestring) and not value:
```

Comment:
Actually you may use `option=True` to enable option, or `option=False` (or even `option=None`) to disable it. `option=''` is used to provide real empty values, like in test that you have modified.